### ISSUE TITLE
Add `logger-objects` to `G` rules documentation

### ISSUE THREAD

**Issue by AbdealiLoKo** [astral-sh/ruff#6764]
I was trying to use the `G` rules in ruff for the first time. I have not used flake8-logging-format in the past

```py
# cat /tmp/try.py
import logging
logging.warning(f'Exception when setting up readline: {"123"}')
logger.warning(f'Exception when setting up readline: {"123"}')
```

On running ruff on this:
```sh
# ruff /tmp/try.py --isolated --select G
/tmp/try.py:2:17: G004 Logging statement uses f-string
/tmp/try.py:3:16: G004 Logging statement uses f-string
Found 2 errors.
```

But when I add a import on top to actually import my `logger` variable:
```py
# cat /tmp/try.py
import logging
from myapp import logger
logging.warning(f'Exception when setting up readline: {"123"}')
logger.warning(f'Exception when setting up readline: {"123"}')
```

`ruff` is not catching the `G004` rule for `logger.warning` anymore.
```sh
# ruff /tmp/try.py --isolated --select G
/tmp/try.py:3:17: G004 Logging statement uses f-string
Found 1 error.
```

From reading previous issues ...
My understanding is that the heuristic assumes that any variable that ends with `logger` should be considered as a `logging.Logger`
But I was not expecting that to change if I import `logger` from another module - this behavior seemed like a bug

tried the versions:
 - python 3.7.16 and 3.10.10
 - ruff 0.0.285

**Comment by charliermarsh** [comment]
Can you try adding my_app.logger to logger-objects? https://beta.ruff.rs/docs/settings/#logger-objects

**Comment by AbdealiLoKo** [comment]
Yep - that got it working 👍 
It was not clear to me that I could configure it :)

Would be nice if that was documented in https://beta.ruff.rs/docs/rules/logging-f-string/ as configurations that can affect this rule ?
